### ISSUE TITLE
Illegal mix of collations  - when clicking the create missing collabs button

### ISSUE THREAD

**Issue by jdragojevic** [pculture/unisubs#2262]
On amara.org environment team.  In the settings/workflows tab, clicked the button for 'create missing collaborations' 

Got this error about the collations.

```
Error message
django.db.utils:DatabaseError: (1267, "Illegal mix of collations (utf8_general_ci,IMPLICIT) and (utf8_unicode_ci,IMPLICIT) for operation '='")
```

`https://rpm.newrelic.com/accounts/207361/applications/3323332/traced_errors/404a31-ac389237-19a4-11e5-8f08-b82a72d2466d`

**Comment by bendk** [comment]
I think the easiest way to fix this is #2242.  However, that ticket depends on a few other issues.  How urgent is this one?

**Comment by jdragojevic** [comment]
Not urgent, since collab model isn't being used like that yet.  I just happened to run across it today.

**Comment by bendk** [comment]
I think this actually should be fixed on production now with my partial fix for #2242.

**Comment by jdragojevic** [comment]
Yes this is fixed for me on production now.
